app-hw-37/src/Store/images.ts

```diff
@@ -0,0 +1,39 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit'
```

Comment:
prefer to get data on the fly, then save it two time
it will reduce count of error in your code

Reply:
I mean -> better to get images from posts reducer, than save it here